### PR TITLE
Add Valeriia Amelchenia profile as a new mentor

### PR DESCRIPTION
**Pull request by nora-weisser**
Added new mentor into the list

Screenshots:

Screenshot 2023-09-17 at 20 54 19
Screenshot 2023-09-17 at 20 54 29
Screenshot 2023-09-17 at 20 54 39

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked the [contributor guide](https://github.com/WomenWhoCode/london/blob/main/CONTRIBUTING.md) 
- [x] I checked include the related website screenshots changes

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -1504,4 +1504,47 @@
...
+    My skill set extends beyond QA, as I took some online courses in 3D graphics creation and further animation in related software (Blender, ZBrush, Artec Studio). I am an active mentor on ADPList platform and share my insights on my own Telegram and YouTube channels.
+    I am characterised by my critical thinking and a constant self-learner with creative approach to tackling intricate challenges.Senior QA engineer with over than 4 years of dedicated experience in maintaining product quality at the highest level. Proficient in establishing effective workflows in QA team and collaborating with the teams of developers, analysts and project management. I have successfully led numerous projects involving the creation of automation tests suites from scratch, for both web and desktop applications (Squish framework on Python and Cypress with Typescript). Additionally, I have led the development of a standalone desktop application for calibrating 3D scanners.
+    <br>
```
Comment (Neha) [inline _data/mentors.yml:1520]: question: As per HTML, the valid tag is `<br />`. On google search for yml I found that people use `<br>`  or`<br/>`.  So, is `<br>` is a valid tag for YML?

**Review by Neha — APPROVED** [review]
LGTM. left a small question

File: _data/mentors.yml (hunk 1 above)
Comment (dricazenck) [reply _data/mentors.yml:1520]: we can add here "anything" as long we have the scape `| ` as Eleonora did.

File: mentors.html [hunk 2]
```diff
@@ -98,6 +98,16 @@ <h5 class="position">{{mentor.position}}</h5>
                                         {% include icons/website.svg %}
                                     </a>
                                     {% endif %}
+                                    {% if network.instagram %}
+                                    <a href="{{network.instagram}}" target="_blank">
+                                        {% include icons/instagram.svg %}
+                                    </a>
+                                    {% endif %}
+                                    {% if network.telegram %}
```
Comment (dricazenck) [inline mentors.html:106]: nice 👍

**Review by dricazenck — APPROVED** [review]

**Pull request merged**